pkgs/development/libraries/quazip/default.nix

```
@@ -12,12 +12,15 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ zlib qtbase ];
-  nativeBuildInputs = [ qmake ];
+  nativeBuildInputs = [ qmake ]
+    ++ stdenv.lib.optional stdenv.isDarwin fixDarwinDylibNames;
+  qmakeFlags = [ "quazip" ]
+    ++ stdenv.lib.optional stdenv.isDarwin [ "LIBS=-lz" ];
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Provides access to ZIP archives from Qt programs";
-    license = stdenv.lib.licenses.gpl2Plus;
+    license = licenses.gpl2Plus;
```

```suggestion
    license = licenses.lgpl21Plus;
```
According to https://github.com/stachenov/quazip/blob/master/COPYING and source headers

---

Updated license